Plugins/AppVeorBuldReport/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Reflection;
+
+[assembly: AssemblyDescription("GitExtensions plugin for integration with Build Server")]
```

Comment:
```suggestion
[assembly: AssemblyDescription("GitExtensions plugin for AppVeyor integration")]
```

Reply:
No, this is in the build server integration, some parts are apparently needed for AppVeyor here - a separate refactoring

Reply:
"AppVeorBuldReport" invalid folder name. Plugin for all types.